src/main/java/org/alliancegenome/curation_api/model/entities/HTPExpressionDatasetSampleAnnotation.java

```diff
@@ -0,0 +1,160 @@
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexingDependency;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.KeywordField;
+
+import com.fasterxml.jackson.annotation.JsonView;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Entity
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@ToString(callSuper = true)
+@Schema(name = "HTPExpressionDatasetSampleAnnotation", description = "POJO that represents the HighThroughputExpressionDatasetSampleAnnotation")
+@AGRCurationSchemaVersion(min = "2.7.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { AuditedObject.class })
+@Table(indexes = {
+	@Index(name = "htpdatasetsampleannotation_htpExpressionSample_index", columnList = "htpExpressionSample_id"),
+	@Index(name = "htpdatasetsampleannotation_dataprovider_index", columnList = "dataprovider_id"),
+	@Index(name = "htpdatasetsampleannotation_createdby_index", columnList = "createdby_id"),
+	@Index(name = "htpdatasetsampleannotation_updatedby_index", columnList = "updatedby_id")
+})
+public class HTPExpressionDatasetSampleAnnotation extends AuditedObject {
+
+	@IndexedEmbedded(includeDepth = 1)
+	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
+	@OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+	@JsonView({ View.FieldsOnly.class })
+	private ExternalDataBaseEntity htpExpressionSample;
+	
+	@IndexedEmbedded(includePaths = {"curie", "name", "secondaryIdentifiers", "synonyms.name", "abbreviation",
+	"curie_keyword", "name_keyword", "secondaryIdentifiers_keyword", "synonyms.name_keyword", "abbreviation_keyword" })
+	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
+	@ManyToOne
+	@JsonView(View.FieldsOnly.class)
+	private OBITerm htpExpressionSampleType;
+
+	@IndexedEmbedded(includePaths = {"curie", "name", "secondaryIdentifiers", "synonyms.name", "abbreviation",
+	"curie_keyword", "name_keyword", "secondaryIdentifiers_keyword", "synonyms.name_keyword", "abbreviation_keyword" })
+	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
+	@ManyToOne
+	@JsonView(View.FieldsAndLists.class)
+	private MMOTerm expressionAssayUsed;
+
+	@IndexedEmbedded(includePaths = {"age", "whenExpressedStageName", "age_keyword", "whenExpressedStageName_keyword"})
+	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
+	@OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+	@JsonView({ View.FieldsOnly.class })
+	private BioSampleAge htpExpressionSampleAge;
+
+	@IndexedEmbedded(includePaths = {"bioSampleAllele.curie","bioSampleAllele.modInternalId","bioSampleAllele.modEntityId","bioSampleAllele.modEntityId_keyword",
+	"bioSampleAllele.modInternalId_keyword","bioSampleAgm.curie","bioSampleAgm.modInternalId","bioSampleAgm.modEntityId","bioSampleAgm.modInternalId_keyword",
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.
<!-- __reviewdog__:ChA5NTc5OTk3NTJjZTBhZDk2EgpjaGVja3N0eWxl -->